DS_Session4/src/molecularMass/MolecularMass.java

```diff
@@ -0,0 +1,102 @@
+package molecularMass;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Stack;
+/**
+ * this is MolecularMass class which is calculate the Mass of Molecular Compounds
+ * 
+ * @author Aman Gautam
+ * Dated - 3/8/2019
+ */
+public class MolecularMass {
+	
+	int mass=0,submass=0;
+	
+	Map<String, Integer> map = new HashMap<String, Integer>();
```

Comment:
Update name here and use self-explanatory variable names.

Reply:
changes applied